easybuild/easyconfigs/q/Qt6/Qt6-6.4.2-GCCcore-12.2.0.eb

```
@@ -0,0 +1,73 @@
+easyblock = 'ConfigureMake'
+
+name = 'Qt6'
+version = '6.4.2'
+
+homepage = 'https://qt.io/'
+description = "Qt is a comprehensive cross-platform C++ application framework."
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+# disabling use of -ftree-vectorize is required to avoid compilation failures on some systems (e.g. Intel Skylake X)
+toolchainopts = {'vectorize': False}
+
+source_urls = [
+    'https://download.qt.io/official_releases/qt/%(version_major_minor)s/%(version)s/single/',
+    'https://download.qt.io/archive/qt/%(version_major_minor)s/%(version)s/single/',
+    'https://download.qt.io/new_archive/qt/%(version_major_minor)s/%(version)s/single/',
+]
+sources = ['qt-everywhere-src-%(version)s.tar.xz']
+checksums = ['689f53e6652da82fccf7c2ab58066787487339f28d1ec66a8765ad357f4976be']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('binutils', '2.39'),
+    ('pkgconf', '1.9.3'),
+    # deps for QtWebEngine
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+    ('gperf', '3.1'),
+    ('Ninja', '1.11.1'),
+    ('Python', '3.10.8', '-bare'),
+    ('re2c', '3.0'),
+]
+
+dependencies = [
+    ('double-conversion', '3.2.1'),
+    ('GLib', '2.75.0'),
+    ('PCRE2', '10.40'),
+    ('libpng', '1.6.38'),
+    ('HarfBuzz', '5.3.1'),
+    ('graphite2', '1.3.14'),
+    # deps for QtWebEngine
+    ('X11', '20221110'),
+    ('fontconfig', '2.14.1'),
+    ('DBus', '1.15.2'),
+    ('libevent', '2.1.12'),
+    ('libGLU', '9.0.2'),
+    ('libjpeg-turbo', '2.1.4'),
+    ('NSS', '3.85'),
+    ('snappy', '1.1.9'),
+    ('JasPer', '4.0.0'),
+    ('bzip2', '1.0.8'),
+    # ('OpenSSL', '1.1', '', SYSTEM),
+    ('ICU', '72.1'),
+]
+
+build_cmd = "cmake --build ."
+install_cmd = "cmake --install ."
+
+# qtwayland fails to build on (some) Centos 7 systems
+configopts = '-release -platform linux-g++'
+configopts += ' -nomake tests -nomake examples -bundled-xcb-xinput -xcb'
+configopts += ' -- -D QT_AVOID_CMAKE_ARCHIVING_API=ON'
+# configopts += ' -qt-libpng -qt-libjpeg'
```

I don't think we actually need a lot of this anymore, with the switch to `cmake`. 
`-nomake tests -nomake examples` is default behaviour.

```suggestion
configopts += ' -DQT_AVOID_CMAKE_ARCHIVING_API=ON'
```